src/cch/service.rs

```diff
@@ -37,23 +37,23 @@ impl CchService {
         loop {
             select! {
                 _ = self.token.cancelled() => {
-                    tracing::debug!("Cancellation received, shutting down cch service");
+                    crate::debug!("Cancellation received, shutting down cch service");
                     break;
                 }
                 command = self.command_receiver.recv() => {
                     match command {
                         None => {
-                            tracing::debug!("Command receiver completed, shutting down tentacle service");
+                            crate::debug!("Command receiver completed, shutting down tentacle service");
                             break;
                         }
                         Some(command) => {
                             let command_name = command.name();
-                            tracing::info!("Process cch command {}", command_name);
+                            crate::info!("Process cch command {}", command_name);
 
                             match self.process_command(command).await {
                                 Ok(_) => {}
                                 Err(err) => {
-                                    tracing::error!("Error processing command {}: {:?}", command_name, err);
```

Comment:
Since tracing support structured logging, we don't have to set format specifier in the first string passed to the macro.

```
crate::error!("Error processing command", command_name, err);
```

seems to be more elegant for me.